proposals/release-versioning-testing-v2.md

```diff
@@ -17,23 +17,36 @@ We revisit release version testing with the following changes in mind:
+| C4            | v1  | v1           | v1         | v1                     | v0        | Simulates scheduler upgrading |
+| C5            | v1  | v1           | v1         | v1                     | v1        | Simulates cell upgrading      |
+
+## BBS running with locket (v1.25.2 -> develop)<a id="sec-2-2" name="sec-2-2"></a>
 
 | Configuration | Locket | BBS | Diego Client | Router (2) | SSH proxy + Auctioneer | Cells (2) | Notes                         |
 |---------------|--------|-----|--------------|------------|------------------------|-----------|-------------------------------|
 | C0            | v0     | v0  | v0           | v0         | v0                     | v0        | Initial configuration         |
 | C1            | v1     | v0  | v0           | v0         | v0                     | v0        | Simulates upgrading diego-api |
 | C2            | v0     | v1  | v0           | v0         | v0                     | v0        |                               |
 | C3            | v1     | v1  | v1           | v0         | v0                     | v0        | Simulates API upgrading       |
-| C3            | v1     | v1  | v1           | v1         | v0                     | v0        | Simulates Router upgrading    |
-| C4            | v1     | v1  | v1           | v1         | v1                     | v0        | Simulates scheduler upgrading |
-| C5            | v1     | v1  | v1           | v1         | v1                     | v1        | Simulates cell upgrading      |
+| C4            | v1     | v1  | v1           | v1         | v0                     | v0        | Simulates Router upgrading    |
+| C5            | v1     | v1  | v1           | v1         | v1                     | v0        | Simulates scheduler upgrading |
+| C6            | v1     | v1  | v1           | v1         | v1                     | v1        | Simulates cell upgrading      |
 
-## Notes<a id="sec-2-1" name="sec-2-1"></a>
+## Notes<a id="sec-2-3" name="sec-2-3"></a>
 
+-   The above configuration have the added benefit of testing the contract between rep and BBS regarding cell presences (newer BBS can still parse old rep cell presence in both Consul or Locket)
 -   Deploy only one instance for all instance groups except Router and Cell to ensure routability to the test app at all times
 -   Cells are to be updated gracefuly (i.e. each cell evacuates before it stops) to ensure routability to the test app at all times
 -   Configurations follow the order of instance group updates as of [this version of cf-deployment](https://github.com/cloudfoundry/cf-deployment/commit/9be2644da8de08540891e24856bbdb88f9a83f67)
--   We have to test both combinations of different versions of BBS and Locket since different versions can exist during a rolling update
+-   We have to test different versions of BBS and Locket since different versions can exist during a rolling update
```

Comment:
I think think this isn't clearly conveying the point that since either locket or bbs could update first, we need to test all combinations of locket and bbs versioning.